task1.md

```diff
@@ -58,10 +82,10 @@ all:
 
 Further documentation for those who are interested to learn more see:
 
-- link1
-- link2
+- [more about inventories](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#collections)
```

Comment:
Same links as above, doubled again, what documentation are we looking for.

Reply:
crap I wonder if the copy paste didn't work right because these shouldn't be the same